### PR TITLE
fixed accel scaling in motors tab

### PR DESCRIPTION
**Pull request by flaviopinzarrone**
Refers to [PR #13026](https://github.com/betaflight/betaflight/pull/13026) in betaflight repo.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by haslinghuis — APPROVED** [review]

**Comment by nerdCopter** [comment]
needs to be somewhat backward compatible.

**Review by Asizon — DISMISSED** [review]

**Comment by nerdCopter** [comment]
4.4.3



4.5 with said code

**Pull request merged**
